src/modules/users/user-list/users.js

```diff
@@ -16,7 +16,6 @@
     function userController($scope, $rootScope, $state, $uibModal, $interval, utils, config, userStore, AuthManager, Notifications) {
```

Comment:
I don't see the use of "utils" service in this controller, please remove the extra dependencies if there is any.

Reply:
Acknowledge